doc/api/http.md

```diff
@@ -151,7 +168,7 @@ added: v0.11.4
 Destroy any sockets that are currently in use by the agent.
 
 It is usually not necessary to do this.  However, if you are using an
-agent with KeepAlive enabled, then it is best to explicitly shut down
+agent with HTTP Keep-Alive enabled, then it is best to explicitly shut down
```

Comment:
this should be `keepAlive` (HTTP Keep-Alive is always used, remember, `Connection: keep-alive` *IS* "HTTP Keep-Alive")